transfer-data/destinations.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: 'Destinations'
+description: 'This page explains how to set up destinations.'
+---
+
+<Note>
+Flow is currently in preview. To try it out, [sign up for a free preview](https://forms.axiom.co/flow).
+</Note>
+
+To transfer and transform data from an Axiom dataset to a destination, you need to set up a destination. This is where data is transferred. Once you set up a destination, you can use it in any flow.
```

Comment:
```suggestion
To transform and transfer data from an Axiom dataset to a destination, you need to set up a destination. This is where data is transferred. Once you set up a destination, you can use it in any flow.
```

Switch of order to reflect common (/only?) pattern of transforming before dispatching.